runtime/dart_vm.cc

```diff
@@ -64,9 +64,8 @@ static const char* kDartLanguageArgs[] = {
     // clang-format on
 };
 
-static const char* kDartPrecompilationArgs[] = {
-    "--precompilation",
-};
+static const char* kDartPrecompilationArgs[] = {"--precompilation",
+                                                "--enable-isolate-groups"};
```

Comment:
@mkustermann Is there a risk to turning this on for users that don't ever call `Dart_CreateIsolateInGroup`?  I worry about affecting the stability of users who don't use these experimental APIs.